src/store/lookup.ts

```diff
@@ -151,7 +153,11 @@ async function handleAdBlock(request: HTTPRequest, adBlockRequestHandler: any) {
  * @param browser Puppeteer browser.
  * @param store Vendor of graphics cards.
  */
-async function lookup(browser: Browser, store: Store) {
+async function lookup(
+  browser: Browser,
+  store: Store,
+  sendNotification: SendNotification
```

Comment:
Most of the added code in this file is passing the `sendNotification` to the functions that need it. 

The rest is sending notifications in lieu of logging